tests/test_fixtures.py

```diff
@@ -0,0 +1,769 @@
+import subprocess
+import logging
+import json
+import tempfile
+import threading
+import os
+
+from os import devnull
+from os import path
+from uuid import uuid4
+from os import urandom
+from functools import wraps
+from http.server import SimpleHTTPRequestHandler, HTTPServer
+
+from fake_http_server.fake_test_server import FakeTestServerBackground
+
+
+logger = logging.getLogger(__name__)
+
+
+class Aktualizr:
+
+    def __init__(self, aktualizr_primary_exe, aktualizr_info_exe, id,
+                 uptane_server, ca, pkey, cert, wait_port=9040, wait_timeout=60, secondary=None, output_logs=True,
+                 run_mode='once', director=None, image_repo=None, **kwargs):
+        self.id = id
+
+        self._aktualizr_primary_exe = aktualizr_primary_exe
+        self._aktualizr_info_exe = aktualizr_info_exe
+        self._storage_dir = tempfile.TemporaryDirectory()
+
+        with open(path.join(self._storage_dir.name, 'secondary_config.json'), 'w+') as secondary_config_file:
+            secondary_cfg = json.loads(Aktualizr.SECONDARY_CONFIG_TEMPLATE.format(port=wait_port, timeout=wait_timeout))
+            json.dump(secondary_cfg, secondary_config_file)
+            self._secondary_config_file = secondary_config_file.name
+
+        with open(path.join(self._storage_dir.name, 'config.toml'), 'w+') as config_file:
+            config_file.write(Aktualizr.CONFIG_TEMPLATE.format(server_url=uptane_server.base_url,
+                                                               ca_path=ca, pkey_path=pkey, cert_path=cert,
+                                                               serial=id[1], hw_ID=id[0],
+                                                               storage_dir=self._storage_dir,
+                                                               db_path=path.join(self._storage_dir.name, 'db.sql'),
```

Comment:
Normally our database is named "sql.db", but that can be overwritten with `storage.sqldb_path`. Not a big deal, but the deviation seems like an opportunity for confusion.

Reply:
yeah, I guess, this is just mistyping.